City-of-Helsinki/helsinki-paatokset
# PP-226: Issue page accessibility fixes

Address accessibility issues on issue pages.

To test:

- Run `drush cr`
- Make sure you have issue and agenda item data on your local
- Find an issue page with related agenda item, eg. https://helsinki-paatokset.docker.so/fi/paatokset-issue/58243
- Check that 'tags' -section is within its own section with role=Region attribute
- Check that the dropdown at the top of the page has links contained within a nav-element and its hover / focus states can be differentiated
- Check that attachment links have and aria-label that communicates that they will open a pdf file
- Navigation at the top of the page should no longer have tabindex=1
- Download decision -button should have better contrast on hover
